src/components/ParticipantProvider/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { createContext } from 'react';
+import { RemoteParticipant } from 'twilio-video';
+import useGridParticipants from '../../hooks/useGridParticipants/useGridParticipants';
+import usePresentationParticipants from '../../hooks/usePresentationParticipants/usePresentationParticipants';
+
+/*
+ *  The hooks used by the VideoProvider component are different than the hooks found in the 'hooks/' directory. The hooks
+ *  in the 'hooks/' directory can be used anywhere in a video application, and they can be used any number of times.
+ *  the hooks in the 'VideoProvider/' directory are intended to be used by the VideoProvider component only. Using these hooks
+ *  elsewhere in the application may cause problems as these hooks should not be used more than once in an application.
+ */
```

Comment:
Not sure about this comment, since the useGridParticipants and usePresentationParticipants hooks are not in the videoProvider/ directory. 

Not sure if we need to move them here though? We could, because developers won't need to ever user them anymore (thanks to this provider), but also there's no harm in them being used elsewhere, which isn't the case with the other hooks like useRoom or useLocalTracks. 

What do you think about where these hooks should live? In any event, it might be good to update this comment to be specific to this component. We could explain why we had to move these hooks to their own provider.